tasks/ember_templates.js

```diff
@@ -37,7 +37,7 @@ function manualCompile(templateCompilerPath, template, grunt){
   vm.runInContext(templateCompilerJs, context, 'ember-template-compiler.js');
 
   // Compile the template
-  vm.runInContext('compiledJS = (module.exports || exports).precompile(template);', context);
+  vm.runInContext('compiledJS = ((typeof Ember !== "undefined" && Ember.HTMLBars) || module.exports || exports).precompile(template);', context);
```

Comment:
Ah, thanks for highlighting that change @zoltan-nz.

Yes, there will be a much more significant delta for Ember 2.9 :)